frontend/src/scenes/surveys/DuplicateToProjectModal.tsx

```diff
@@ -49,9 +49,15 @@ export function DuplicateToProjectModal(): JSX.Element {
                     </LemonButton>
                     <LemonButton
                         type="primary"
-                        onClick={() => duplicateToProject({ sourceSurvey: survey, targetTeamId: selectedProject?.id })}
+                        onClick={() => {
+                            if (selectedTeam?.id === currentTeam?.id) {
+                                return duplicateSurvey()
+                            }
+
+                            duplicateToProject({ sourceSurvey: survey, targetTeamId: selectedTeam?.id })
+                        }}
                         loading={duplicatedToProjectSurveyLoading}
-                        disabledReason={!selectedProject ? 'Select a project' : undefined}
+                        disabledReason={!selectedTeam ? 'Select a project' : undefined}
```

Comment:
this feels wrong, should it be Select a team instead?

Reply:
in the UI we call then projects, but in the code we call it team, which is why I only changed the variables